src/components/SupportIndicator/SupportIndicatorUpdated.tsx

```diff
@@ -0,0 +1,81 @@
+import { Grid, Typography, LinearProgress, Box } from '@mui/material';
+import { withStyles } from '@mui/styles';
+import React from 'react';
+import './SupportIndicator.scss';
+
+const BorderLinearProgress = withStyles({
+  root: {
+    height: 20,
+    borderRadius: 10,
+  },
+  bar: {
+    borderRadius: 10,
+  },
+})(LinearProgress);
+
+/*
+ * A generic UI element to display support for a claim/by a source in 5 steps from low to high
+ */
+export interface SupportIndicatorProps {
+  /* The number of supporting/disputing/balanced sources */
+  supporting: number;
+  disputing: number;
+  balanced: number;
+  labels: string[];
+}
+
+export const SupportIndicatorUpdated: React.FC<SupportIndicatorProps> = ({
```

Comment:
I do not see that you are using this component anywhere. Let's remove it if we are not using it.

Reply:
Was this file deleted? Or we are using it?

Reply:
It's deleted.